.github/scripts/generate_manifests.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Script for the initial generation of manifest.yaml files in inputs directories
+
+if [[ $# -ne 3 ]]; then
+  echo "Error, arguments required are not given."
+  exit 1
+fi
+
+yamf_module_path="${1}"
+yamf_module_name="${2}"
+start_dir="${3:-.}"
+
+module use "$yamf_module_path"
+module load "$yamf_module_name"
+
+dir_without_subdir=$(find "$start_dir" -type d -links 2 ! -empty)
```

Comment:
Again a comment would be good to explain what this is doing.

Reply:
Commented in https://github.com/ACCESS-NRI/model-config-inputs/pull/5/commits/4760f32ab35b22cc9532f37d03afb8eb8e4e3bff